integration_test/third_party_apps_test/main_test.go

```diff
@@ -722,14 +722,14 @@ func modifiedFiles(t *testing.T) []string {
 // means we should test all applications.
 func isCriticalFile(f string) bool {
 	if strings.HasPrefix(f, "submodules/") ||
-		strings.HasPrefix(f, "integration_test/third_party_apps_data/agent/") {
+		strings.HasPrefix(f, "integration_test/third_party_apps_test/") {
```

Comment:
this line is wrong, we should just delete it. The `agent` directory was deleted at some point.

Reply:
Done